lib/guessgame.rb

```diff
@@ -0,0 +1,27 @@
+# Challenge 4 - Guessing Game
+
+operation = gets.chomp
```

Comment:
This challenge requires you to check if the guessed number is correct or not. If the guess is incorrect, you should ask the question again.

You could repeat this by having a loop implemented in your prompt.